client/views/helpers/qr_code/qr_code.html

```diff
@@ -0,0 +1,15 @@
+<template name="qr_code">
```

Comment:
@singhsanket143 can you fix the style of this template?

Reply:
@peter-hank Shall I update the indentation of this file?